xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamFFmpeg.cpp

```diff
@@ -155,3 +139,10 @@ std::string CDVDInputStreamFFmpeg::GetFileName()
   }
   return CDVDInputStream::GetFileName();
 }
+
+void CDVDInputStreamFFmpeg::SetRealtime(bool realtime)
```

Comment:
the assumptions you take here are wrong i.e. realtime streams can be paused or seeked back if timeshift is active